gettingStarted.html

```diff
@@ -19,6 +20,8 @@
             integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
             crossorigin="anonymous"
         ></script>
+        <script src="js/common.js"></script>
```

Comment:
We should figure out if this displays on mobile. For example, we may want to check the `orientationchange` event. (Just a comment, this doesn't have to be in this PR)

Reply:
I tested this with the device simulator on edge so it should work, though haven't checked on actual devices or different browsers.